scripts/perftest/python/utils_fs.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+import os
+from os.path import join
+import glob
+from functools import reduce
+from utils_exec import subprocess_exec
+
+# Utility support for all file system related operations
+
+
+def create_dir_local(directory):
+    """
+    Create a directory in the local fs
+
+    directory: String
+    Location to create a directory
+    """
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+
+def write_success(time, cwd):
```

Comment:
I thought `cwd` is mostly to specify the current working directory.
How about naming this variable to something else, like `directory` or `dir` or something similar?